src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+  width: 100%;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid $white;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin: 0 0 16px;
+  }
+
+  &__rev {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+
+  &__rev-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main;
+    margin: 0;
```

Comment:
1. please set margin 0 for all p elements instead of give this property to every class of p element 
2. you can create file normalize scss in which you write all such styles like *, :root, body, p etc. -> it's a good practice to reset such default styles of browsers in separate file and set some general settings there